srcpkgs/NetworkManager/template

```diff
@@ -68,6 +68,12 @@ pre_configure() {
 	fi
 }
 
+do_check() {
+	if [ "$XBPS_CHECK_PKGS" = full ]; then
+		ninja -C build test
+	fi
+}
```

Comment:
You can just set `make_check=extended` in the template.

Reply:
Fixed.